mypyc/test-data/irbuild-set.test

```diff
@@ -202,11 +202,12 @@ def update(s, x):
     s :: set
     x :: list
     r0 :: int32
-    r1, r2 :: None
+    r1 :: None
+    r2 :: None
```

Comment:
Although it's not something that important, I wonder how this happened.

Reply:
This seems worth investigating. The underlying issue causing might be important.